com.vscodium.codium-insiders.yaml

```diff
@@ -0,0 +1,143 @@
+id: com.vscodium.codium-insiders
+sdk: org.freedesktop.Sdk
+runtime: org.freedesktop.Sdk
+runtime-version: '23.08'
+command: codium-insiders
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=ssh-auth
+  - --share=network
+  - --device=all
+  - --filesystem=host
+  - --filesystem=xdg-config/gtk-3.0:ro
+  - --persist=.vscode-oss
+  - --allow=devel
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.freedesktop.Flatpak
```

Comment:
This is not allowed by default. You need an exception for this

Reply:
indeed, com.vscodium.codium has those exceptions and codium-insiders will need the same since it's an IDE that need to work with host services and binaries

Reply:
I'll look how is now com.visualstudio.code change this since this was taken from them

Reply:
com.visualstudio.code use --talk-name=org.freedesktop.Flatpak, but replace `flatpak-spawn` with [host-spawn](https://github.com/1player/host-spawn), do you think will be a good if I implement the same?

Reply:
Your choice, I don't know about that project.

Reply:
You didn't request an exception for this, without that the build won't pass https://github.com/flathub/flatpak-builder-lint/pulls